lib/phoenix/test/channel_test.ex

```diff
@@ -239,9 +239,12 @@ defmodule Phoenix.ChannelTest do
   """
   defmacro connect(handler, params) do
     if endpoint = Module.get_attribute(__CALLER__.module, :endpoint) do
+      stringified_params =
+        Code.eval_quoted(params) |> elem(0) |> stringify_params |> Macro.escape
```

Comment:
One option is to call it `__stringify__` and add a `@doc false`. This way it won't be imported nor show up in docs. It can be used both by macros and regular functions.